### DISCUSSION TITLE
Exe is trying to delete somehting in Program Files/Windows (Antivirus is blocking)

### DISCUSSION THREAD

**Discussion by Ignis231** [THGSCST/Vices-and-Sins-MOD#10] [Q&A] [answered]
The executable included keeps trying to delete something in the listed folder
C:\ProgramData\Microsoft\Windows\WER\Temp
any idea what this is or why?

**Comment by THGSCST** [comment, marked as answer]
Did your game crashed? The WER (Windows Error Reporting) Service is designed to collect debug details about software failures.

According to [VirusTotal.com behavior analysis](https://www.virustotal.com/gui/file/70955c70e472d1eaed09157b12c5a01bd4efe15b3f8526ff15f4a73a37955b56/behavior), the exe can drop this files:
```
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1DE3.tmp
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1DE3.tmp.WERInternalMetadata.xml
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1DE4.tmp
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1DE4.tmp.WERInternalMetadata.xml
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1E8F.tmp
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1E8F.tmp.csv
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1E90.tmp
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1E90.tmp.csv
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1EDF.tmp
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1EDF.tmp.txt
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1EE0.tmp
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1EE0.tmp.txt
C:\Windows\System32\spp\store\2.0\cache\cache.dat
C:\Windows\System32\spp\store\2.0\data.dat.tmp
```


Then, delete this:
```
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1DE3.tmp.WERInternalMetadata.xml
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1DE4.tmp.WERInternalMetadata.xml
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1E8F.tmp.csv
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1E90.tmp.csv
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1EDF.tmp.txt
C:\ProgramData\Microsoft\Windows\WER\Temp\WER1EE0.tmp.txt
C:\Windows\System32\spp\store\2.0\cache\cache.dat
```
This behavior seems OS dependent. I used ProcessMonitor and didn't see these actions on my machine, my game didn't crash either. The changes I made to the exe was just disabling the cheats and enabling some flags. You can see all changes using a hex editor comparing the original with my modified exe.

**Comment by Ignis231** [comment]
Hi,
Yeah it's happening whenever my game crashes.
Given the files are in a temp folder I don't think it matters that much.
